inst/tutorials/introduction/introduction.Rmd

```diff
@@ -0,0 +1,265 @@
+By so doing, we can also adjust individual parameters manually outside of the `get_parameters()` function.
+
+```{r parameters_alternative, exercise=TRUE}
+parameters <- get_parameters()
+parameters$human_population <- 1000
+parameters$human_population
+```
+
+
+`malariasimulation` also contains various helper functions and in-built parameters to help you design your parameter set. We will not cover these in detail here, but they include setting:
+
+- population demographies `set_demography()`,
+- mosquito species and proportions `set_species()`, with in-built species specific parameter sets: An. arabiensis `arab_params()`, An. funestus `fun_params()`, An. gambiae `gamb_params()` and An. stephensi `steph_params()`,
+- drug interventions including MDA and SMC `set_drugs()`, `set_clinical_treatment()`, `set_mda()`, `set_smc()`,
+- bednet and IRS vector control `set_bednets()`, `set_spraying()`,
+- pre-erythrocytic and transmission blocking vaccines `set_mass_pev()`, `set_pev_epi()`, `set_tbv()`.
+
+Please see the package vignettes for more details on how to use these, or ask a demonstrator!
+
+
+
+
+## 2: Set equilibrium
+
+Do you remember seeing default parameters for the initial human disease state proportions and human immunity levels? These may not be applicable to the scenario you are trying to simulate (e.g., particularly if you are starting a simulation with no immunity to malaria in a high-transmission malaria-endemic area!).
+
+Luckily, we have a function that sets the initial transmission level AND will calculate an equilibrium for that transmissibility level and the chosen model parameters: `set_equilibrium()`. In `set_equilibrium()` we define the initial EIR (entomological inoculation rate - is the rate of human biting multiplied by the sporozoite infection rate), which can be varied to reflect the disease prevalence in a region.
+
+```{r set_eq, exercise=TRUE}
+# Generate parameters
+parameters <- get_parameters()
+
+# Set equilibrium
+parameters_eq <- set_equilibrium(parameters = parameters, init_EIR = 5)
+```
+
+Note that the parameters list now contains a new element called `init_EIR`:
+
+```{r set_eq_check, exercise=TRUE}
+parameters <- get_parameters()
+parameters_eq <- set_equilibrium(parameters = parameters, init_EIR = 5)
+parameters_eq$init_EIR
+```
+
+Try setting the initial EIR to 10.
+
+```{r set_eq_exercise, exercise=TRUE}
+
+```
+
+
+
+## 3: Run simulation
+
+Once the parameters have been set, we can run a simulation.
+
+The main simulation function is `run_simulation()`, which requires the parameter set and the number of timesteps in days you would like to run the simulation for. It can be helpful to define what a year is!
+
+```{r run_sim, exercise=TRUE}
+# Generate parameters
+parameters <- get_parameters()
+
+# Set equilibrium
+parameters_eq <- set_equilibrium(parameters = parameters, init_EIR = 5)
+
+# Run simulation
+year <- 365
+simulation_output <- run_simulation(timesteps = 10 * year, parameters = parameters_eq)
```

Comment:
Can we add something like this, otherwise it looks like nothing really happens when you hit the run code button

```suggestion
simulation_output <- run_simulation(timesteps = 10 * year, parameters = parameters_eq)

# Have a look at the raw model output
head(simulation_output)
```

Reply:
Oh - I see that is exactly where you go in the next section! Up to you the best approach :)